applications/machine_learning/configuration/nrf5340dk_nrf5340_cpuapp/prj.conf

```diff
@@ -112,6 +112,7 @@ CONFIG_NEWLIB_LIBC_FLOAT_PRINTF=y
 CONFIG_MCUMGR=y
```

Comment:
Consider using `thingy53: samples and applications:` in commit message.
It was proposed by Torsten to mark commits that affect thingy53 samples and apps to make them easier to identify in history.